marklogic-client-api/src/test/java/com/marklogic/client/test/RowManagerTest.java

```diff
@@ -200,6 +200,12 @@ public static void beforeClass() throws IOException, InterruptedException {
   }
   @AfterClass
   public static void afterClass() {
+    DocumentManager docMgr = Common.client.newDocumentManager();
```

Comment:
I've found the best practice for tests is for data to be deleted before a test begins and not deleted after a test ends. Leaving the test data in place makes debugging a lot easier when the test fails. And it's often useful to run a test to generate data and then fiddle with it in qconsole. 

I know it can be difficult to switch to that practice when an existing test suite doesn't already delete data at the start of the tests. But given that this test class was already creating data and not deleting it (which I think is good), I would test to see if it's fine to leave these documents around. If they break some other test, then that test should just clear out the database before it begins.